tests/lbs/test_lbs.sh

```diff
@@ -38,21 +40,22 @@ fi
 
 # Check that an unclean disconnect is handled appropriately
 printf "Testing LBS disconnection cleanup..."
-( $TESTLBS $SOCK & ) 2>/dev/null
-sleep 0.1 && killall test_lbs
+( $TESTLBS $SOCK & echo $! > $TESTLBS_PID ) 2>/dev/null
+$MSLEEP 100 && kill "$(cat $TESTLBS_PID)"
```

Comment:
What do you think about this?  Theoretically, if testlbs already exited, the OS could have already reused that PID for a different process, so we'd be killing something at random.

I think that's sufficiently unlikely... and I'm not even certain how I'd eliminate the possibility entirely.  I mean, I could check with `ps` to ensure that the command-line of that pid is still `test_lbs`, but theoretically the process could stop by itself after checking, and be reused before I kill that number.